framework_tvb/tvb/tests/framework/core/factory.py

```diff
@@ -41,22 +41,23 @@
 import os
 import random
 import tvb_data
```

Comment:
@VinczeRobert From the history of commits you seem to have worked a bit on this Factory class. 
Pls have a quick look at this file refactor ... and the code could be even further improved ;-)